sdcm/utils/adaptive_timeouts/load_info_store.py

```diff
@@ -122,6 +122,9 @@ def cpu_load_5(self) -> float:
     def shards_count(self) -> int:
         return len([key for key in self._get_scylla_metrics() if key.startswith('scylla_lsa_free_space')])
 
+    def scylla_scheduler_shares(self) -> list:
```

Comment:
see `_get_scylla_metrics` has cached output with ttl=60 seconds.
If we need this updated on demand, better use `self._get_metrics(port=9180)` or decrease ttl to smthing smaller (e.g. 5 seconds should be fine)

Reply:
done